### PR TITLE
Announcements: Make strings translatable

### PR DESCRIPTION
**Pull request by AnnikaCodes**
Unfortunately, I don't speak any of the languages PS! is translated into well enough to actually do any translation, but I did handle the technical side.

One minor snag (which I want to discuss either here or on Discord) is that `Chat#plural` only correctly creates a plural form for English, and not other languages. Personally, I think if we are to be truly multilingual, assuming that words are always plural is probably simpler (albeit less clean-looking) than trying to programmatically generate plural forms for every language PS is translated into or gets a translation for in the future.

### CODE REVIEW

File: translations/dutch.json [hunk 1]
```diff
@@ -131,6 +131,19 @@
...
 		"You are already blocking challenges!": "Je blokkeert al uitdagingen!",
 		"You are already available for challenges!": "Je bent al beschikbaar voor uitdagingen!",
-		"You are available for challenges from now on.": "Je bent vanaf nu beschikbaar voor uitdagingen."
+		"You are available for challenges from now on.": "Je bent vanaf nu beschikbaar voor uitdagingen.",
+
+		"The announcement has ended.": "",
+		"Battles do not support announcements.": "",
+		"You are not allowed to use filtered words in announcements.": "",
+		"There is already a poll or announcement in progress in this room.": "",
+		"An announcement was started by ${user.name}.": "",
```
Comment (Zarel) [inline translations/dutch.json:140]: Is this how translations are usually implemented?

Comment (Zarel) [reply translations/dutch.json:140]: Looks like yes. Interesting.

**Comment by Zarel** [comment]
Looks like a straight improvement, so sure.

File: translations/dutch.json (hunk 1 above)
Comment (AnnikaCodes) [reply translations/dutch.json:140]: Yeah, I try to make any variables in template strings I'm going to translate descriptive enough that translators can understand with minimal technical knowledge.

**Pull request merged**